tokens/src/core/components/Dropdown.json

```diff
@@ -37,7 +37,13 @@
           "base": { "source": "$dropdown-padding-y", "$value": ".5rem" },
           "item": { "source": "$dropdown-item-padding-y", "$value": ".25rem" }
         },
-        "header": { "source": "$dropdown-header-padding", "$value": "{spacing.dropdown.padding.y.base} {spacing.dropdown.padding.x.item}" }
+        "header": {
+          "source": "$dropdown-header-padding",
+          "$value": {
+            "x": "{spacing.dropdown.padding.x.item}",
+            "y": "{spacing.dropdown.padding.y.base}"
+          }
```

Comment:
Having `x.item` as `x` and `y.base` as `y` seems a little odd to me. We have both `base` and `item` in `x` and `y` for the general `padding` token right above this, should we do that here too?

Reply:
Probably yes, since we had the SCSS variable [$dropdown-header-padding](https://github.com/openedx/paragon/blob/1a6b9b33cf09423a9fe760c3f2c0615acc8f31ef/src/Dropdown/_variables.scss#L33) before